Cognite.Simulator.Utils/RetryPolicy.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using CogniteSdk;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Polly;
+using Polly.Extensions.Http;
+namespace Cognite.Simulator.Utils
+{
+    /// <summary>
+    /// Configuration for HTTP client policies.
+    /// </summary>
+    public static class HttpClientPolicyConfiguration
+    {
+        
+        /// <summary>
+        /// Gets or sets the action to execute when a transient HTTP error occurs.
+        /// </summary>
+        public static Action<Exception, TimeSpan, int, Context, IServiceProvider> OnRetry { get; set; } = (exception, timeSpan, retryCount, context, serviceProvider) => { 
```

Comment:
There might not be need of this, we might able to use the extractor utils to do this: https://github.com/cognitedata/dotnet-extractor-utils/blob/67442743d275d1e8393c79a49275b990593285a7/Cognite.Extensions/CogniteUtils.cs#L551C57-L551C71